test/e2e/scheduling/priorities.go

```diff
@@ -148,6 +167,12 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		framework.ExpectNoError(err)
 		err = e2epod.WaitForPodsRunningReady(cs, metav1.NamespaceSystem, int32(systemPodsNo), 0, framework.PodReadyBeforeTimeout, map[string]string{})
 		framework.ExpectNoError(err)
+
+		// skip if the most utilized node has less than the cri-o minMemLimit available
+		// otherwise we will not be able to run the test pod once all nodes are balanced
+		if nodesAreTooUtilized(cs, nodeList) {
```

Comment:
This additional check seems based on cri-o limitation.
Is it fine to apply the same limitation for environments which runtimes are different from cri-o?

/cc @oomichi

Reply:
This is a good question, and I would appreciate if any runtimes folks could chime in. I imagine most runtimes will have some sort of similar minimums, but even finding the cri-o minimum was difficult for me. I'm not sure who to tag for that

Reply:
@mrunalp , would you be able to help us figure out how `minMemoryLimit` should be handled for runtimes other than cri-o? I see you originally introduced it to cri-o (in https://github.com/cri-o/cri-o/pull/1723), so hoping you could shed some light on this issue

Reply:
@oomichi I am still trying to reach someone more familiar with different runtimes, but from my own assessment I don't think there is any risk to other runtimes for adding this change. At most, the test will be using more resources than necessary on runtimes with smaller minimums (but it is a serial test, so that isn't a problem)

However, without this change the test is flakey on CRI-O, which needs to be supported by k8s. So I think that is justification for needing this

Reply:
from my (brief) investigation, containerd does not add a similar check.

CRI-O added the check because we found 12MB was a good minimum to allow all containers in the pod to be created. runc itself takes 3-4MB, so in a sufficiently large pod (two containers, plus infra container), container creates were being killed.

I agree with @damemi, at worse it uses more resources than necessary (on the order of maybe 8 MB per pod)